.github/workflows/veriy-minimal-parachain-dev-spec-chains.yml

```diff
@@ -0,0 +1,137 @@
+name: veriy-minimal-parachain-dev-spec-chains.yml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  setup:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        template:
+          - package_name: 'parachain-template-runtime'
+            runtime_path: './templates/parachain/runtime'
+            artifact_name: 'wasm-parachain-template-runtime'
+            chain_spec_output: 'parachain_test_chain_spec'
+            runtime_wasm_path: 'parachain-template-runtime/parachain_template_runtime.compact.compressed.wasm'
+          - package_name: 'minimal-template-runtime'
+            runtime_path: './templates/minimal/runtime'
+            artifact_name: 'wasm-minimal-template-runtime'
+            chain_spec_output: 'minimal_test_chain_spec'
+            runtime_wasm_path: 'minimal-template-runtime/minimal_template_runtime.compact.compressed.wasm'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
```

Comment:
Please install rust as it's done [here](https://github.com/paritytech/polkadot-sdk/blob/2dd2bb5a847d7bf006493d9fe9b10a13289118b6/.github/workflows/checks-quick.yml#L191-L202) (taking toolchain version from `.github/env`)